GSA/made-in-america
# MIA-184: Content/urgent waiver

## Description

- includes new urgent-waiver directory
- includes an urgent waiver
- updates urgent waiver page to include link to the new waiver

## How to test

- navigate to `about/urgent-waivers/`
- click first bullet item(date)
- ensure `.xlsx` is downloaded
